nextflow.config

```diff
@@ -10,41 +10,37 @@
 params {
     // Input options
     input                        = null
+    peptide_col_name             = 'sequence'
     peptides_split_maxchunks     = 100
     peptides_split_minchunksize  = 5000
     split_by_variants            = false
     split_by_variants_size       = 0
     split_by_variants_distance   = 110000
+    wide_format_output           = false
 
     // References
     genome_reference = 'grch37'
 
     // Options: Predictions
-    tools                   = 'syfpeithi'
-    tool_thresholds         = null
-    use_affinity_thresholds = false
+    tools = 'mhcflurry'
 
-    // Options: Filtering
-    filter_self           = false
-    show_supported_models = false
+    // Options: Filter variants by user-defined proteome
+    proteome_reference = null
 
     // External tools
     external_tools_meta = "${projectDir}/assets/external_tools_meta.json"
     netmhc_system       = 'linux'
     netmhcpan_path      = null
-    netmhc_path         = null
     netmhciipan_path    = null
-    netmhcii_path       = null
 
     // Options: Peptides
-    max_peptide_length        = 11
-    min_peptide_length        = 8
-    max_peptide_length_class2 = 16
-    min_peptide_length_class2 = 15
+    min_peptide_length_classI  = 8
+    max_peptide_length_classI  = 12
+    min_peptide_length_classII = 8
+    max_peptide_length_classII = 25
 
     // Options: Annotation
-    proteome  = null
-    wild_type = false
+    wild_type = true
```

Comment:
Do we want that to be default?

Reply:
I found that information really useful! And its not really compute / request heavy afaik

Reply:
OK, might be different now.